wordpress-mobile/WordPress-Android
# Issue/8700 update layouts large

### Fix
Update the ***Quick Start*** layouts on the ***Sites*** tab (https://github.com/wordpress-mobile/WordPress-Android/issues/8642) and full-screen dialog (https://github.com/wordpress-mobile/WordPress-Android/issues/8643) for large screens as described in https://github.com/wordpress-mobile/WordPress-Android/issues/8700.  See the screenshots below for illustration.

![v2_tablet_sites](https://user-images.githubusercontent.com/3827611/50059451-e907e480-0154-11e9-8572-bc580d4ad23a.png)

![v2_tablet_list](https://user-images.githubusercontent.com/3827611/50191026-bba86b80-02f9-11e9-89b6-bbb7cb8fb99c.png)

#### Note
There are a couple differences between the **_Design_** and **_Develop_** screenshots.  In the **_Sites_** screenshots, the row icons and text are vertically aligned with the start of the icons and text in the **_Switch Site_** and **_Next Steps_** views.  In the full-screen dialog screenshots, the toolbar color matches the updated designs shown in the [full-screen dialog changes](https://github.com/wordpress-mobile/WordPress-Android/pull/8720) and the **_Customize Your Site_** type is used since the **_Create Your Site_** task is not the **_Grow Your Audience_** type and to show a similar layout to the design mockup.

### Test
Both small and large screen devices should be used to test these changes to ensure the screenshots in the [full-screen dialog changes](https://github.com/wordpress-mobile/WordPress-Android/pull/8720) are shown for small screens and the screenshots above are shown for large screens.

0. Select site with Quick Start in progress.
1. Go to ***Sites*** tab.
2. Notice ***Next Steps*** view as shown above.
3. Tap **_Create Your Site_** or **_Grow Your Audience_** type.
4. Notice completed and uncompleted lists as shown above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.
